api/http-request-practice.md

```diff
@@ -162,45 +133,10 @@ What happens?
 
 To keep things simple, for this example, we just load the same origin, worldtimeapi.org, so we didn't have to worry about it.
 
-### `http.request` in Node
-
-There are too many HTTP libraries to count in Node, but there is one that is built-in that we'll use.
-
-```javascript
-const options = {
-  hostname: 'worldtimeapi.org',
-  port: 443,
-  path: '/api/ip',
-  method: 'GET'
-};
-
-const req = https.request(options, res => {
-  console.log(`statusCode: ${res.statusCode}`)
-
-  res.on('data', d => {
-    process.stdout.write(d)
-  })
-});
-
-req.end();
-
-```
-
-### Guided Practice
```

Comment:
Why remove Guided Practice?

Reply:
It was of the deleted section

Reply:
Please put it back, it is actually a separate section.